src/app/page/admin/users/user/admin.user.component.html

```diff
@@ -35,36 +32,17 @@ <h1 class="font-color-white header-text">User info: </h1>
         <img src="https://en.opensuse.org/images/0/0b/Icon-user.png" class="user-image">
       </div>
       <div class="row">
-        <div *ngIf="(user.accountStatus == 'ACTIVE' || user.accountStatus == 'BLOCKED') ">
-          <button class="btn btn-secondary btn-wide" data-toggle="modal" data-target="#confirmModal"
-                  (click)="toggleUp()">
+        <div *ngIf="(user.accountStatus == 'ACTIVE' || user.accountStatus == 'BLOCKED')">
```

Comment:
Why did you chech if the user is active or blocked? Can you simply check if the user is blocked?